nautobot/circuits/urls.py

```diff
@@ -1,41 +1,21 @@
 from django.urls import path
 
+from nautobot.core.views.routers import NautobotUIViewSetRouter
 from nautobot.dcim.views import CableCreateView, PathTraceView
 from nautobot.extras.views import ObjectChangeLogView, ObjectNotesView
 from . import views
 from .models import Circuit, CircuitTermination, CircuitType, Provider, ProviderNetwork
 
+
 app_name = "circuits"
+router = NautobotUIViewSetRouter()
+router.register("providers", views.ProviderUIViewSet, basename="provider")
+router.register("provider-networks", views.ProviderNetworkUIViewSet, basename="providernetwork")
+router.register("circuit-types", views.CircuitTypeUIViewSet, basename="circuittype")
+router.register("circuits", views.CircuitUIViewSet, basename="circuit")
+router.register("circuit-terminations", views.CircuitTerminationUIViewSet, basename="circuittermination")
```

Comment:
I don't think that the `basename` argument here is necessary for all of these, since it defaults to the model's name in lower case.
```suggestion
router.register("providers", views.ProviderUIViewSet
router.register("provider-networks", views.ProviderNetworkUIViewSet)
router.register("circuit-types", views.CircuitTypeUIViewSet)
router.register("circuits", views.CircuitUIViewSet)
router.register("circuit-terminations", views.CircuitTerminationUIViewSet)
```